README.md

```diff
@@ -103,154 +105,56 @@ The process of selecting a best value involves the following steps:
 
-However, the browser will also contain bids that are targeted at Interest Groups. These bids cannot escape the Fledge sandbox, which means that logic to choose between them must be sent into Fledge.
-
-There are two parts to the logic that must be sent into TurtleDove:
-1) Prebid IG controller - the script that runs the overall IG bid selection process
-2) Prebig IG SSP adapters - the SSP implemented functionality to choose from among an SSP's IG bids
-
-## Prebid IG SSP Adapter
-In the same way that an SSP implements an adapter to integrate with Prebid.js, an SSP will need to implement an adapter that selects from the IG based bids that it owns. This entails implementing some of the logic noted above:
-
-* Bid targeting - which bids are eligible to participate based on advertiser targeting?
-* Context targeting - which bids are eligible to participate based on pub restrictions?
-* Valuation - does the current context influence the value of these bids? Are there Deals or Private Marketplace rules in effect?
-
-In order to do this selection process the SSP adapter needs to have the contextual information passed in, including the publisher's restrictions and any active business models. The IG bids also need to contain any advertiser restrictions so that those can be evaluated given the current context. In the past all this information stayed within the SSP but for ad selection to be completed in Fledge the information must be sent to the browser.
-
-## Prebid IG Controller
-The Prebid IG controller must receive the winning contextual bid (already chosen by Prebid.js) and all the SSP specific metadata. The controller passes the SSP metadata to the SSP IG adapter and receives a bid (or none) from the SSP adapter. The IG Controller then compares the results from all the SSP IG Adapters and chooses a winner. Finally, the IG Controller compares the contextual winner to the IG winner. If an IG winner exists and is higher than the contextual winner, the result of ```navigator.runAdAuction``` is not ```false``` Prebid.js will render the IG ad by passing the returned opaque object to a fenced frame. If the contextual winner has the higher value then ```false``` is returned and the overall Prebid.js script renders the contextual ad.
+However, the browser will also contain user interest group memberships which could increase the value of the ad to a buyer. The Parakeet server will carefully select which IGs can be used in server side ad selection to ensure user privacy.
 
+The Parakeet server will only make an ad request to a single ad tech partner for each ad slot. Therefore, publishers who wish to work with multiple SSPs will need to leverage a server to fan-out requests to each SSP and choose the most appropriate bid to compete with the ad server chosen contextual ad. Prebid server already performs a similar operation and could be extended to handle this responsibiliy.
 ## Prebid IG Flow
```

Comment:
```suggestion
## Prebid.js IG Flow

```

Reply:
this flow describes both prebid server and prebid.js